content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -1,65 +1,66 @@
+===============================
 
-With Odoo Sales, it is possible to add or modify optional products directly on quotations
-(under the **Optional Products** tab, as you can see below).
+With the Odoo *Sales* application, it is possible to add or modify optional products directly on
+quotations (located under the :guilabel:`Optional Products` tab on a quotation form, in the *Sales*
+application).
 
 .. image:: optional_products/optional_products_2.png
    :align: center
-   :class: img-thumbnail
    :alt: How to add optional products to your quotations on Odoo Sales
 
 .. note::
-   By clicking on the **Customer Preview** button, you will be able to see what your customers
-   will have as possibilities after opening a quotation from their received email.
+   Clicking the :guilabel:`Customer Preview` button, reveals a preview of the quotation customers
+   receive via email, along with the optional products they have a chance of adding to their order.
 
    .. image:: optional_products/optional_products_3.png
       :align: center
-      :class: img-thumbnail
       :alt: Preview your quotations on Odoo Sales
 
-In practice, your customers will be able to add different optional products to their order by
-using associated carts, with a user-friendly layout. More than that, if a customer selects all the
-optional products suggested, these additional items will automatically fill in the quotation
-managed by the salesman.
+In practice, your customers are able to add different optional products to an order by clicking the
+:guilabel:`Shopping Cart icon`, located to the right of the optional product line.
+
+Additionally, if a customer selects all the optional products suggested, these additional items are
+automatically filled in the quotation managed by the salesman.
 
 .. image:: optional_products/optional_products_4.png
    :align: center
-   :class: img-thumbnail
    :alt: How to select optional products on Odoo Sales
 
-Like this, salespeople will see each movement made by the customer and tracking the order will be
-all the better.
+When the customer adds an optional product(s) to an order, the salesperson sees each movement made
+by the customer, providing them with accurate tracking of the entire order in the backend of the
+*Sales* application (on the specific quotation form).
 
 .. image:: optional_products/optional_products_5.png
    :align: center
-   :class: img-thumbnail
    :alt: How to follow each movement made by your customers on Odoo Sales
 
-Add optional products to your quotation templates
-=================================================
+Optional products on quotation templates
+========================================
 
-Be sure to check out our documentation about :doc:`quote_template` to understand how you can
-enable, create, design and manage your own quotation templates before reading this part.
+Be sure to review the documentation about :doc:`quote_template` to better understand how quotation
```

Comment:
Please use global relative paths rather than local relative paths
```suggestion
Be sure to review the documentation about
:doc:`/applications/sales/sales/send_quotations/quote_template` to better understand how quotation
```